spec/models/sequence_spec.rb

```diff
@@ -102,10 +102,14 @@
 
     let(:simple_portal_hash) do
       url = "http://test.host#{Rails.application.routes.url_helpers.sequence_path(sequence)}"
+      author_url = "http://test.host/sequences/#{sequence.id}/edit"
+      print_url = "http://test.host/sequences/#{sequence.id}/print_blank"
```

Comment:
Yeah I thought about that, but I followed the pattern here: https://github.com/concord-consortium/lara/blame/master/spec/models/lightweight_activity_spec.rb#L330

I figured @knowuh had a reason for doing it that way.  But now that I look at it, I don't see an advantage. I'll probably change all three places unless I can think of a benefit to this literal approach.